prombench/Makefile

```diff
@@ -1,4 +1,5 @@
 INFRA_CMD        ?= ../infra/infra
+PROVIDER 		 ?= gke
```

Comment:
Will be nice to have this in a GKE specific PR like mentioned above. (We can add the use of `PROVIDER` in the Makefile aswell) this means the GKE specific PR would also change the location of the gke files in it.